homeassistant/components/spotify/__init__.py

```diff
@@ -58,12 +58,17 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if DOMAIN in config:
         async_create_issue(
             hass,
-            DOMAIN,
-            "removed_yaml",
+            HOMEASSISTANT_DOMAIN,
```

Comment:
Should not the whole `async_setup` be removed and change `CONFIG_SCHEMA` to `cv.config_entry_only_config_schema()` instead?

Don't think `removed_yaml` should ever appear as it should indeed be removed instead?

Reply:
Hmm, good point 👍